game/resource/English/ability/units/boss/tooltip_as_passive_6.txt

```diff
@@ -0,0 +1,8 @@
+//=============================================================================
+// Simple Boss Desolate
+//=============================================================================
+"DOTA_Tooltip_ability_boss_as_passive_6"                      "Attack Speed Bonus"
+"DOTA_Tooltip_ability_boss_as_passive_6_Lore"                  "Roshan has been using this since 6.86."
+"DOTA_Tooltip_ability_boss_as_passive_6_Description"                "This unit's increased attack speed offsets its low base attack time. You can't slow its attacks as easily."
+"DOTA_Tooltip_ability_boss_as_passive_6_radius"            "RADIUS:"
```

Comment:
I have approximately 0.023 of an idea what I'm doing and this is how Return handles potentially turning into an aura